docs/jwst/cube_build/main.rst

```diff
@@ -73,88 +65,71 @@ We use the following terminology to define the spectral range divisions of MIRI:
+created from more than one band of data. If the IFU cube have a non-linear wavelength dimension
+there will be an added binary extension table to the output fits IFU cube. This extension has
+the label, WCS-TABLE, and contains the wavelength for each of the IFU cube wavelength planes. This table follows the
+FITs standard described in, *Representations of spectral coordinates in FITS*, Greisen, et al., **A & A**  446, 747-771, 2006. 
+
+The input data to ``cube_build`` can take a variety of forms, including a single file, a data
 model passed from another pipeline step, a list of files in an association table, or a collection of exposures in a
 data model container (ModelContainer) passed in by the user or from a preceding pipeline step. Because the MIRI IFUs
 project data from two channels onto a single detector, choices can or must be made as to which parts of the input data
 to use when constructing the output cube even in the simplest case of a single input image. The default behavior
 varies according to the context in which ``cube_build`` is being run.
 
-In the case of the ``calwebb_spec2`` pipeline,
-for example, where the input is a single MIRI or NIRSpec IFU exposure, the default output cube will be built from
-all the data in that single exposure. For MIRI this means using the data from both channels (e.g. 1A and 2A) that
-are recorded in a single exposure. For NIRSpec this means using data from the single grating+filter combination
-contained in the exposure.
+In the case of the ``calwebb_spec2`` pipeline, for example, where the input is a single MIRI or NIRSpec IFU exposure,
+the default output cube will be built from all the data in that single exposure. For MIRI this means using the data
+from both channels (e.g. 1A and 2A) that are recorded in a single exposure and the output IFU cube will have a 
+non-linear wavelength dimension. For NIRSpec the data  is from the single grating and filter combination contained
+in the exposure and will have a linear wavelength dimension. The calwebb_spec2 pipeline calls cube_build 
+with ``output_type=multi``.
 
 In the ``calwebb_spec3`` pipeline, on the other hand, where the input can be a collection of data from multiple
 exposures covering multiple bands, the default behavior is to create a set of single-band cubes. For MIRI, for
 example, this can mean separate cubes for bands 1A, 2A, 3A, 4A, 1B, 2B, ..., 3C, 4C, depending on what's included in
 the input. For NIRSpec this may mean multiple cubes, one for each grating+filter combination contained in the
-input collection.
+input collection. The calwebb_spec3 pipeline calls cube_build with ``output_type=band``. These types of IFU cubes will have
+a linear-wavelength dimension. If the user wants to combine all the data together covering several band they can using
+the option ``output_type=multi` and the resulting IFU cubes will have a non-linear wavelength dimension. 
```

Comment:
Need a second ` at the end of multi. This may be what's causing the doc build to fail.